src/ol_openedx_canvas_integration/ol_openedx_canvas_integration/__init__.py

```diff
@@ -2,6 +2,6 @@
 This is an integration of canvas with edX.
 """
 
-__version__ = "0.5.1"
+__version__ = "0.5.3"
```

Comment:
The definitive location for version is in the pyproject.toml https://github.com/mitodl/open-edx-plugins/blob/main/src/ol_openedx_canvas_integration/pyproject.toml#L3

Once you update it there this is ready to merge.

Reply:
@blarghmatey I just caught the typo in pyproject.toml. Fixed it.